fuel-types/README.md

```diff
@@ -0,0 +1,15 @@
+# Fuel Types
+
+[![build](https://github.com/FuelLabs/fuel-types/actions/workflows/ci.yml/badge.svg)](https://github.com/FuelLabs/fuel-types/actions/workflows/ci.yml)
```

Comment:
The links in these badges refer to the old repos - might be worth checking all the READMEs for links that need updating.

Could also be worth adding a markdown linter like in the Sway or fuelup repos, but that might be best done in a follow-up PR.